index.html

```diff
@@ -0,0 +1,484 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers | Creative Design Agency</title>
+    <meta name="description" content="Avengers, a Creative Design Agency based out of San Fransisco California, speicalizing in Web Design, UI Design, Icon Design, Branding & Logotype">
```

Comment:
Make sure to use properly encoded entities; `&amp;` instead of `&` (ampersands can cause trouble). Read more here: https://mathiasbynens.be/notes/ambiguous-ampersands